parser/scanner.l

```diff
@@ -85,30 +82,6 @@ func (l *lexer) Expr() ast.ExprNode {
 	return l.expr
 }
 
-func (l *lexer) Inj() int {
-	return l.inj
-}
-
-func (l *lexer) SetInj(inj int) {
-	l.inj = inj
-}
-
-func (l *lexer) SetPrepare() {
-	l.prepare = true
-}
-
-func (l *lexer) IsPrepare() bool {
```

Comment:
Only prepare statement allows placeholder. So how can we check if a placeholder if valid without this?